kncaputo/rancid-tomatillos
# Feature/initial components

## What's this PR do?
- Adds folders and files for components
- Adds initial functionality for components (MovieDetails displays but is unfinished)

## How should this be manually tested?
- Test by running `npm start` from command line
- Click a movie and see that the movie cards disappear (and `MovieContainer` component) and `MovieDetails` react component is visible

## Any background context you want to provide?
-Does not yet utilize server requests

## Screenshots/GIFs
<img width="1160" alt="Screen Shot 2020-12-03 at 4 43 19 PM" src="https://user-images.githubusercontent.com/68157759/101091785-ada39200-3586-11eb-8bb0-2ff276ee06d1.png">
